src/jdk.compiler/share/classes/com/sun/tools/javac/resources/compiler_zh_CN.properties

```diff
@@ -93,7 +93,7 @@ compiler.err.abstract.cant.be.instantiated={0}是抽象的; 无法实例化
 compiler.err.abstract.meth.cant.have.body=抽象方法不能有主体
 
 # 0: kind name, 1: symbol
-compiler.err.already.annotated={0} {1}已进行注释
+compiler.err.already.annotated={0} {1} 已进行批注
```

Comment:
Noticeably, _zh_CN_ has updates to a few values that were not updated in the original, but are a result of updated translations by the translation tool. For this particular case it looks like a space is added, I'm not sure if this can be attributed to localization rules or not.